### PR TITLE
feat(core): temporarily add page mode editor settings to appearance settings

### PR DESCRIPTION
**Pull request by JimmFly**
Will remove it when editor settings is released.

CleanShot 2024-09-04 at 18 19 29@2x

### CODE REVIEW

**Comment by JimmFly** [comment]
* **#8090** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8090?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @JimmFly and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by EYHN — APPROVED** [review]

File: packages/frontend/core/src/components/affine/setting-modal/general-setting/editor/edgeless/note.tsx [hunk 1]
```diff
@@ -201,7 +206,7 @@ export const NoteSettings = () => {
           items={shadowItems}
           trigger={
             <MenuTrigger className={menuTrigger}>
-              {String(settings['affine:note'].edgeless.style.shadowType)}
+              {currentShadow?.[0]}
```
Comment (akumatus) [inline packages/frontend/core/src/components/affine/setting-modal/general-setting/editor/edgeless/note.tsx]: Import `NoteShadowMap` to use. Already modfied in following PR.

https://app.graphite.dev/github/pr/toeverything/AFFiNE/8094

File: packages/frontend/core/src/components/affine/setting-modal/general-setting/editor/edgeless/pen.tsx [hunk 2]
```diff
@@ -4,32 +4,51 @@ import { EditorSettingService } from '@affine/core/modules/editor-settting';
...
+      return settings.brush.color;
+    } else {
+      return resolvedTheme === 'dark' ? LineColor.White : LineColor.Black;
+    }
+  }, [resolvedTheme, settings.brush.color]);
+
+  const currentColor = useMemo(() => {
+    return Object.entries(LineColor).find(([, value]) => value === color);
+  }, [color]);
+
```
Comment (akumatus) [inline packages/frontend/core/src/components/affine/setting-modal/general-setting/editor/edgeless/pen.tsx:25]: Better to use `LineColorMap[value]` to get color name.

File: packages/frontend/core/src/components/affine/setting-modal/general-setting/editor/edgeless/pen.tsx [hunk 3]
```diff
@@ -2,34 +2,55 @@ import { MenuItem, MenuTrigger, Slider } from '@affine/component';
...
   const settings = useLiveData(editorSetting.settings$);
+  const getColorFromMap = useColor();
+
+  const color = useMemo(() => {
+    if (typeof settings.brush.color === 'string') {
+      return settings.brush.color;
+    } else {
+      return resolvedTheme === 'dark' ? LineColor.White : LineColor.Black;
+    }
+  }, [resolvedTheme, settings.brush.color]);
```
Comment (akumatus) [inline packages/frontend/core/src/components/affine/setting-modal/general-setting/editor/edgeless/pen.tsx]: Please remove this useless color code.

**Review by akumatus — APPROVED** [review]

**Pull request merged**